viewing_party/party.py

```diff
@@ -0,0 +1,255 @@
+#Wave 1
+def create_movie(title,genre,rating):
+    if title and genre and rating:
+        movie_dict = {"title": title, 
+        "genre": genre,
+         "rating": rating,
+        }
+        return movie_dict
+    else:
+        return None
+
+
+
+def add_to_watched(user_data, movie):
+    #user_data = {watched: [{}]}
+    #watched = user_data["watched"]
+    #watched.append(movie)
+    #return user_data
+    # user_data["watched"] = []
+    # movie = { "title": "Title A",
+    # "genre": "Horror", 
+    # "rating": 3.5}
+    if movie:
+        user_data["watched"].append(movie)
+    else:
+        user_data["watched"] = []
+    return user_data
```

Comment:
```suggestion
    if movie:
        user_data["watched"].append(movie)
    return user_data
```

The `else` clause here is optional. If `movie` is falsy, then there'd be to change to the watched list in `user_data`.